Khan/perseus
# Ensure axis lines and arrowheads are 2px thick.

## Summary:
The default thickness for lines in Mafs is 1px. Our designs indicate 2px. These styling updates ensure that axis lines and arrowheads have the correct thickness.


Issue: LEMS-1929

## Test plan:
1. Launch storybook for Perseus (`yarn start`)
1. View the [Interactive Graph editor with locked lines story](http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph-editor--with-locked-lines)
1. Inspect the graph parts (hard to see visually, so inspecting is best for verifying)
   * Axis lines should have a `stroke-width` of 2px.
   * Arrowheads on lines and rays should also have a `stroke-width` of 2px.

## Affected visuals (may need to squint to see the difference):

### Before
![Line Thickness - Before](https://github.com/Khan/perseus/assets/13896410/b4ae9ee6-ff5f-4bc0-ab73-745e0beb6be0)

### After
![Line Thickness - After](https://github.com/Khan/perseus/assets/13896410/c0961441-72a2-49d7-aa0d-feb5dd2d3ff5)
